### PR TITLE
hide some options in creat class modal

### PR DESCRIPTION
**Pull request by smallst**
fix ENG-487
now creating:
image
and editing:
image

### CODE REVIEW

**Review by mrfinch — COMMENTED** [review]
can you remove classroom announcement field too likewise?

**Comment by nwinter** [comment]
Looking at it now, it occurs to me that showing just the four options at first, but having a small "More Options" link at the bottom that would expand and show the rest of them could be good. (We would reorder items & gems + classroom code autocomplete to be below the first four options that we do show by default.)

**Comment by smallst** [comment]
image

File: ozaria/site/components/teacher-dashboard/modals/ModalEditClass.vue [hunk 1]
```diff
@@ -81,6 +82,9 @@ export default Vue.extend({
       getSessionsMapForClassroom: 'levelSessions/getSessionsMapForClassroom',
       courses: 'courses/sorted'
     }),
+    moreOptionsText () {
+      return this.moreOptions ? 'Less Options' : 'More Options'
```
Comment (mrfinch) [inline ozaria/site/components/teacher-dashboard/modals/ModalEditClass.vue:86]: It should be `Hide Options`

Also, i18n the text.

If it's not too much effort, can we add a caret up and down symbol next to text?

**Pull request merged**